pom.xml

```diff
@@ -35,6 +35,12 @@
 
 
     <dependencies>
+        <!-- Please don't move logging further down - order of dependencies does matter-->
```

Comment:
Similar to zebedee - please explain why and how to avoid conflicts in future.

Reply:
Same answer as for zebedee. I ll add this to the maintenance documentation.

Order of the dependencies in the dep section does matter to the maven build system. Restolino for example requires dp-logging and will load its own dp-logging version when dp-logging is not loaded previously. Resolino particular uses and older version of slf4j as we only did minor version updates.    

We found that dp-logging had a library missing org.slf4j which required all java applications to add this dependency so that they would work. What we did is we added this dependency in dp-logging to be pulled in from there and avoid having to include it in every application. However, restolino is also using org.slf4j but an older version of slf4j-api since we are only doing minor updates. By changing the order of dependencies and putting dp-logging above restolino the build then worked as it's using version 2.0.10 which is being pulled from dp-logging.

Reply:
Yes - please add some context to the comment in the code.

Reply:
Ok. will do.